backend/src/modules/player-profiles/core/application/command/CreatePlayerProfile.ts

```diff
@@ -0,0 +1,21 @@
+export class CreatePlayerProfile {
+    readonly playerId: string;
+    readonly firstName: string;
+    readonly lastName: string;
+    readonly phoneNumber: string;
+    readonly emailAddress: string;
+
+    constructor(props: {
+        playerId: string;
+        firstName: string;
+        lastName: string;
+        emailAddress: string;
+        phoneNumber: string;
+    }) {
+        this.playerId = props.playerId;
+        this.firstName = props.firstName;
+        this.lastName = props.lastName;
+        this.emailAddress = props.emailAddress;
+        this.phoneNumber = props.phoneNumber;
+    }
+}
```

Comment:
@nowakprojects tutaj rozumiem, że ten plik ma tak wyglądać, tak?
Czemu pytam - dokładnie to samo istnieje w domain w pliku PlayerProfile, ale to co wyżej jest potrzebne JEDYNIE tam gdzie występuje nazwa commandHandlers - czyli PlayersProfilesModuleCore --> commandHandlers + do pliku CreatePlayerProfileCommandHandler. Tak jak wspominałeś, w tym przypadku po prostu zbiegło się, że commnad wygląda tak samo jak obiekt domenowy, Si?